server/kv.go

```diff
@@ -393,25 +381,26 @@ func (store store) EnsureRSAKey() (rsaKey *rsa.PrivateKey, returnErr error) {
 
 func (store store) StoreOneTimeSecret(token, secret string) error {
 	// Expire in 15 minutes
-	appErr := store.plugin.API.KVSetWithExpiry(
-		hashkey(prefixOneTimeSecret, token), []byte(secret), 15*60)
-	if appErr != nil {
-		return errors.WithMessage(appErr, "failed to store one-ttime secret "+token)
+	_, err := store.plugin.client.KV.Set(
+		hashkey(prefixOneTimeSecret, token), []byte(secret), pluginapi.SetExpiry(15*60))
+	if err != nil {
+		return errors.WithMessage(err, "failed to store one-ttime secret "+token)
 	}
 	return nil
 }
 
 func (store store) LoadOneTimeSecret(key string) (string, error) {
-	b, appErr := store.plugin.API.KVGet(hashkey(prefixOneTimeSecret, key))
-	if appErr != nil {
-		return "", errors.WithMessage(appErr, "failed to load one-time secret "+key)
+	var secret string
+	err := store.get(hashkey(prefixOneTimeSecret, key), secret)
```

Comment:
That looks suspicious. The stored data is not in JSON format, but `store.get` will try to unmarshal it. Are you sure this line works?

Reply:
Nice catch. Yes, this looks like the code was accidentally over-written. It should read:

```suggestions
err := store.plugin.client.KV.Get(hashkey(prefixOneTimeSecret, key), secret)
```

I will push a commit to fix this.